Portal/View/UIKit/PullToRefreshable.swift

```diff
@@ -16,7 +16,7 @@ internal protocol PullToRefreshable {
     
     var mailbox: Mailbox<ActionType> { get }
     
-    func configure(pullToRefresh properties: RefreshProperties<ActionType>)
+    func configure(pullToRefresh properties: RefreshProperties<ActionType>, offset: CGPoint)
```

Comment:
Why are you adding an `offset` parameter and not using it?

Reply:
ups I forgot delete it! srry